components/core/src/clp_s/TimestampPattern.cpp

```diff
@@ -71,7 +73,7 @@ append_padded_value_notz(int value, char padding_character, size_t max_length, s
  * @return true if conversion succeeds, false otherwise
  */
 static bool convert_string_to_number(
-        string const& str,
+        std::string_view str,
```

Comment:
sorry I missed this one.
nit: this file has been using "using std::string". how about we do the same for std::string_view, so we get rid of the std:: and it's consistent with the rest of file?

Reply:
Looking a bit more at the file it looks like its already a bit inconsistent with using std:: even for things it declared as `using`. I'll go through and remove all of the using to make things consistent.

Reply:
Actually on second though it mostly isn't using std::, so I'll just change the few other exceptions to be consistent.